docs/source/tutorial-pyfunc.rst

```diff
@@ -0,0 +1,368 @@
+.. _tutorial_pyfunc:
+
+Store and Load Custom Models Using `mlflow.pyfunc`
+==================================================
+
+When you save your Python project in MLflow with ``log_model``, the particularl "flavor" of MLflow you're using will store the model in the appropriate format. For instance, ``mlflow.keras.log_model`` will save your Keras model as an HDF5 file.
+
+The most general and flexible flavor of the MLflow Model format is the **"pyfunc"** format. This "Python Function" format allows you to wrap a customized Python function for inferencing. Using pyfunc also allows you full customization of your model initialization, data preparation, inferencing, reporting, and so forth, all of which may be attractive even if you are using a more specific flavor of MLflow.
+
+This tutorial will show how to save and load models in the pyfunc format. It will also show how to serve an MLflow model as a REST endpoint and how to deploy a model in a Docker container.
+
+.. _setup:
+
+Setup
+-----
+
+This tutorial assumes that you have installed and used the MLflow library for tracking and managing machine learning experiments. At the very least, you should have run an example that used ``log_model`` to save a model in the MLflow Model format. The :ref:`quickstart-mlops` is a good place to start. 
+
+The final section of this tutorial will show how to deploy a model in a Docker container. To run the Docker commands, you will need to have Docker installed. If you do not have Docker installed, you can skip the final section of this tutorial.
+
+- If you are not already running an MLflow Tracking Server, start one by running ``mlflow server`` in a terminal. This will start an MLflow Tracking Server on port 5000. (For customization, see :ref:`tutorial-tracking`.)
+
+- Set the ``MLFLOW_TRACKING_URI`` environment variable to the URI of the server. 
+
+.. important:: 
+    You **must** set the ``MLFLOW_TRACKING_URI`` environment variable to the URI of the server. If MLflow sees that the tracking URI is not an HTTP or HTTPS URI, it will not allow you to save or load models.
```

Comment:
```suggestion
    You **must** set the ``MLFLOW_TRACKING_URI`` environment variable to the URI of the MLflow tracking server. If MLflow sees that the tracking URI is not an HTTP or HTTPS URI, it will not allow you to save or load models.
```